corehq/apps/smsforms/models.py

```diff
@@ -296,7 +296,7 @@ def claim_channel_for_session(cls, session):
         """
         channel = session.get_channel()
         with cls._critical_section(channel):
-            if cls.channel_is_available_for_session(session):
+            if cls._channel_is_available_for_session(session) or cls._clear_stale_channel_claim(channel):
```

Comment:
Looks like `_channel_is_available_for_session` calls `_clear_stale_channel_claim`. Why call it again on this line?

Reply:
Thanks for pointing that out, you're right that was an oversight. I added this line before realizing I should move it down into `_channel_is_available_for_session`, and ended up leaving it in both. (The blackbox behavior is still the same so tests didn't fail, but you're right that it's entirely redundant.)